utils.js

```diff
@@ -56,6 +56,21 @@ function getRoot(list) {
   return root;
 }
 
+function toChecksumAddress(address) {
```

Comment:
is this function can be made more simply?

Reply:
This time it's easier, you can display checksum address on the front end

> is this function can be made more simply?